### PR TITLE
Add Clojure (Luminus) Guide

### PR DESCRIPTION
**Pull request by unicodeveloper**
CleanShot 2024-11-06 at 17 41 29@2x

### CODE REVIEW

File: src/docs/guides/luminus.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        ```bash
+        railway init
+        ```
+    - Follow the prompts to name your project.
+    - After the project is created, click the provided link to view it in your browser.
+3. **Add a Postgres Database Service**:
+    - Run `railway add -d postgres`.
+    - Hit **Enter** to add it to your project.
+    - A database service will be added to your Railway project.
+4. **Add a Service and Environment Variable**:
```
Comment (brody192) [inline src/docs/guides/luminus.md:120]: I can't get over how much better this new feature makes the cli guides

File: src/docs/guides/luminus.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+Railway offers multiple ways to deploy your Clojure app, depending on your setup and preference. 
+
+### One-Click Deploy from a Template
+
+If you’re looking for the fastest way to get started, the one-click deploy option is ideal.
+
+Click the button below to begin:
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/DsDYI2)
```
Comment (brody192) [inline src/docs/guides/luminus.md:99]: Thoughts on renaming the service from `clojureluminus` to `Clojure Luminus` or just `Luminus`?

File: src/docs/guides/luminus.md (hunk 1 above)
Comment (unicodeveloper) [reply src/docs/guides/luminus.md:120]: It's amazing. Thank you @Milo123459

File: src/docs/guides/luminus.md (hunk 2 above)
Comment (unicodeveloper) [reply src/docs/guides/luminus.md:99]: `Clojure Luminus` makes sense. Will do

Comment (unicodeveloper) [reply src/docs/guides/luminus.md:99]: Done

**Pull request merged**